src/app/components/utility/input/input.component.html

```diff
@@ -1,11 +1,20 @@
 <div class="input-group">
-  <div *ngIf="type!='file'">    
-  <input class="input-field" (input)="validateInput($event.target.value)" [class.isValid]="isValid" [class.theme-dark]="theme=='dark'" type="{{type}}" [value]="value" [readonly]="isReadonly">
-  <span class="input-bar"></span>
-  <label [class.is-not-empty]="!isEmpty" [class.theme-dark]="theme=='dark'">{{placeholder}}</label>
-  <img class="input-icon" src="{{icon}}" *ngIf="isIconPresent" />
-  <div class="input-message" #textValue [class.hidden]="!(((isRequired && isEmpty)||(!isValid && !isEmpty))  && isDirty)">{{message}}</div>
+  <div *ngIf="type!='file' && type!='textarea'">
+    <input class="input-field" (input)="validateInput($event.target.value)" [class.isValid]="isValid" [class.theme-dark]="theme=='dark'" type="{{type}}" [value]="value" [readonly]="isReadonly">
+    <span class="input-bar"></span>
+    <label [class.is-not-empty]="!isEmpty" [class.theme-dark]="theme=='dark'">{{placeholder}}</label>
+    <i class="input-icon {{icon}}" *ngIf="isIconPresent"></i>
+    <div class="input-message" #textValue [class.hidden]="!(((isRequired && isEmpty)||(!isValid && !isEmpty))  && isDirty)">{{message}}</div>
+  </div>
+
+  <div *ngIf="type=='textarea'">
+    <textarea autosize class="input-field" (input)="validateInput($event.target.value)" [class.isValid]="isValid" [class.theme-dark]="theme=='dark'" [value]="value" [readonly]="isReadonly"></textarea>
```

Comment:
Please try to do this without adding new package. If it is required in multiple places or needed for other features which is not present in Angular  then it is fine but for small improvement adding new package is not a good idea.

Reply:
@Shekharrajak I used this package thinking that it would be good if we have 'text-area' in other places in the future.

Reply:
Cool! Using more packages can slow down the website. Can you check whether it is behaving properly in all the device size or not?

Reply:
@Shekharrajak yeah it's working perfectly, please check if it's working on your side too